### ISSUE TITLE
Localized Asset fields display global values within Vizy

### ISSUE THREAD

**Issue by sidedwards** [verbb/vizy#101]
## 🐛 Bug description
When an Asset that contains localized fields is referenced within a Vizy field, the localized Asset values are not displayed, instead, it displays the most recently saved values.

**Additional info**
- Plugin version: `1.0.9`
- Craft version: `3.7.30.1`
- Multi-site: Yes

## 👣 Reproduction steps
1. Set up Craft for multi-site.
2. Create an Asset field type with localized fields within.
3. Add the Asset field to a Vizy block.
4. Within an Entry, add the Asset field within a Vizy block, click into the Asset, update the localized field values, and save the Asset.
5. Switch to another Site for the Entry.
6. See error. The Asset's values show the most recent changes from the previous site instead of the localized values.

## 🦄 Expected behavior
Assets within Vizy blocks should display their localized values instead of global values.

## 🐴 Actual behavior
Assets within Vizy blocks display global values instead of their localized values.

## 📚 Resources

https://user-images.githubusercontent.com/11489608/153460539-8adfdeda-8d04-4e53-8596-4c0e836ae37f.mp4

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.9",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.10](https://github.com/verbb/vizy/releases/tag/1.0.10)
